docs/docs/channels/email/resend.md

```diff
@@ -31,7 +31,7 @@ Resend allows you to authenticate your sender identity using [Domain Authenticat
 - Visit the [Integrations](https://web.novu.co/integrations) page on Novu.
 - Locate Resend and click on the **Connect** button.
 - Enter your Resend API Key.
-- Fill the `From email address` field using the authenticated email from the previous step.
+- If you have verified your domain in previous step, fill the `From email address` field with any domain email address. For example, if your organization domain name is `example.com` then you can any use any valid email address like `hello@example.com`.
```

Comment:
ref:- https://discord.com/channels/895029566685462578/937309213036785744/1126276110548742154

Reply:
Hey @jainpawan21 is there any specific reason not to use Cloudinary links for images. We use it for all our quickstart guides. 🙂

Reply:
Also run the doc file through grammarly once please!

Reply:
Also run it through grammarly once please!

Reply:
Unfortunately, due to free plan limit, I could not be added in cloudinary. As we are going to do version 2 in docs, I am thinking to manage images better there. 
I checked the text with Grammarly.
let me know if you think I need to update anything grammarwise